funnel/views/space.py

```diff
@@ -38,7 +38,8 @@ def space_data(space):
 
 
 # Test endpoint
-@app.route('/form', methods=['GET', 'POST'], subdomain='<profile>')
+@app.route('/<profile>/form', methods=['GET', 'POST'])
+@funnelapp.route('/form', methods=['GET', 'POST'], subdomain='<profile>')
```

Comment:
Remove this endpoint. We added it to test Baseframe's form generator. It's not required anymore.